src/Cameras/Inputs/arcRotateCameraPointersInput.ts

```diff
@@ -58,6 +58,14 @@ export class ArcRotateCameraPointersInput extends BaseCameraPointersInput {
     @serialize()
     public pinchDeltaPercentage = 0;
 
+    /**
+     * useNaturalPinchZoom will be used instead of pinchPrecision if set to true.

```

Comment:
I'd rephrase this comment (and others in this review) to be something like this:

When useNaturalPinchZoom is true, multitouch zoom will zoom in such that any object in the plane at the camera's target point will scale perfectly with finger motion.  Overrides pinchDeltaPercentage and pinchPrecision.